### DISCUSSION TITLE
Not getting costs for Tesla Superchargers

### DISCUSSION THREAD

**Discussion by Gr3ytooth** [teslamate-org/teslamate#3315] [General]
Sorry for what is probably a stupid question.

I've set up Teslamate and in all but one aspect it is working beatuiful. The problem is that it can't seem to get the costs for my charges at Tesla Superchargers. I've had Teslamate installed since February this year and it has all my charges logged, both DC and AC, but no costs for Tesla SuC. Is this not supposed to be automatically imported from my Tesla account?

My installation is done from docker compose. I'm currently running version Teslamate version 1.27.3 and typically update as soon as I find a new version. My car is connected using Streaming API and as mentioned earlier the connection must be working as it is getting all the drives and charges imported. Just no costs for Tesla SuC. 

Not sure if that is relevant, but I use TeslaMateAgile to get charging costs at home from Tibber. 

Any tips and pointers to why this not is working would be greatly appreciated.

**Comment by cwanja** [comment]
> Is this not supposed to be automatically imported from my Tesla account?

In TeslaMate's current version, it does not pull this information automatically.It has been mentioned and requested several times - mostly recently I can find being here #1831.

@adriankumpf - might be the "next best feature".

PS - no stupid questions.

**Reply by Gr3ytooth** [reply]
Thanks for the reply.

Okey, that's a shame it's not implemented. It would be a really nice feature.
